src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/schema_validation.py

```diff
@@ -10,9 +10,14 @@
 
 LOGGER = configure_logger(__name__)
 
+NOTIFICATION_PROPS = {
```

Comment:
Could you update the documentation to explain how this should be setup too?

See: https://github.com/awslabs/aws-deployment-framework/blob/master/docs/user-guide.md#params

Reply:
Added in guidance as part of the user guide and admin guide